network/p2p/libp2pNode.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/rs/zerolog"
 
 	"github.com/onflow/flow-go/module"
-	netwk "github.com/onflow/flow-go/network"
+	network2 "github.com/onflow/flow-go/network"
```

Comment:
Maybe `libp2pnet` and `flownet` for these rather than `network` and `network2`?

Reply:
applied in https://github.com/onflow/flow-go/pull/183/commits/1305edd953183c139a5bb39ff2bd59a74e96d959